dioptas/widgets/plot_widgets/HistogramLUTItem.py

```diff
@@ -307,9 +308,12 @@ def _configurationButtonClicked(self):
         widget = ColormapPopup(parent=self.scene().views()[0])
 
         widget.setCurrentGradient(self.gradient.saveState())
+        if isinstance(self.imageItem, NormalizedImageItem):
```

Comment:
It shouldn't be needed, but it keeps it compatible with pyqtgraph's `ImageItem` in case I overlooked something.